script/Arcueid/Day 7/07_00_ARC07_3G.txt

```diff
@@ -886,7 +886,7 @@ Most psychics use their powers without knowing the risks,
 [sha:e904f915b7cef97109cb893f6c067ec4a8f999ac]{
 -- Page 491, Offset 17791.
 -- ……まあ、人間でありながら人間の社会では生きられない存在的不適合者だから、その方が幸せなのかもしれないけど」
-...Though, given that they're destined to be outcasts from human society despite being human themselves,
+...Though, given that they were destined to be outcasts from human society despite being human themselves,
```

Comment:
I think the 'potential' form (`are`) makes more sense than `were` here since I don't think we're talking about a specific instance of a person in the past, but making a general statement that continues to be true.